libraries/rib-base/src/main/kotlin/com/uber/rib/core/Interactor.kt

```diff
@@ -42,9 +44,21 @@ public abstract class Interactor<P : Any, R : Router<*>>() : InteractorType, Rib
   @Inject public lateinit var injectedPresenter: P
 
   @CoreFriendModuleApi public var actualPresenter: P? = null
-  private val _lifecycleFlow = MutableSharedFlow<InteractorEvent>(1, 0, BufferOverflow.DROP_OLDEST)
-  public open val lifecycleFlow: SharedFlow<InteractorEvent>
-    get() = _lifecycleFlow
+
+  private val useStateFlow = RibEvents.useStateFlowInteractorEvent
```

Comment:
Hm, we can probably save an allocation here. Let's just use `get() = RibEvents.useStateFlowInteractorEvent`